### ISSUE TITLE
Keras CNN architecture for pneumothorax detection

### ISSUE THREAD

**Issue by serginavarroguimera** [keras-team/keras#14079]
Hi,

We are working on a project in the University and we are developing a CNN architecture for pneumothorax diagnosis. We have first started with the following very simple architecture, but the fact is that the accuracy get's stuck at very exact values. By this I mean that, bearing in mind that we are working with a relatively large dataset (13k dicom images), I don't think it is normal that the accuracy get's always stuck to 0.70000 or 0.60000 and so on. I leave the code below, and we would be very pleased if someone gave us some feedback on it. 

import numpy as np 
import os
import skimage.io as io
import skimage.transform as trans
import numpy as np
from keras.models import *
from keras.layers import *
from keras.optimizers import *
from keras.callbacks import ModelCheckpoint, LearningRateScheduler
from keras import backend as keras
from keras.layers import Dropout
import cv2
import os
import sys
import math
import mxnet as mx
from mxnet import nd, autograd, gluon
from keras.layers.normalization import BatchNormalization

def defModel3(input_shape = (256,256,1)):

    X_input = Input(input_shape)
            
    X = BatchNormalization()(X_input)
    
    X = Conv2D(16, (3, 3),activation='relu',padding='same')(X) 
        
    X = AveragePooling2D((2, 2))(X)  
    
    X = Conv2D(32, (3, 3),activation='relu',padding='same')(X)  
        
    X = AveragePooling2D((2, 2))(X)  
    
    X = Conv2D(64, (3, 3),activation='relu',padding='same')(X) 
    
    X = AveragePooling2D((2, 2))(X)  

    X = Conv2D(128, (3, 3),activation='relu',padding='same')(X)  
        
    X = AveragePooling2D((2, 2))(X)  
    
    # X = Conv2D(256, (2, 2))(X)  

    X = Flatten()(X) 
            
    X = Dense(256, activation='relu')(X)  

    X = Dense(128, activation='relu')(X) 
    
    X = Dense(64, activation='relu')(X)  

    X = Dense(32, activation='relu')(X) 

    X = Dense(2, activation='softmax')(X) 

    # ------------------------------------------------------------------------------

    model = Model(inputs=X_input, outputs=X, name='Model')
    
    opt = SGD(lr=0.00003)
    
    model.compile(optimizer=opt, loss='categorical_crossentropy', metrics=['accuracy'])
        
    return model

Thank you

**Comment by sushreebarsa** [comment]
@serginavarroguimera  Moving this issue to closed status as there has been no recent activity, in case you still face the error please create a new issue.Thanks!
